packages/office-addin-unit-test/src/officeJSMock.ts

```diff
@@ -0,0 +1,150 @@
+import { usageDataObject } from "./defaults";
+
+export class OfficeJSMock {
+  constructor(object?: OfficeObject) {
```

Comment:
We should probably do a better job and the formal method headers . . . I think that aids in intellisense.

Reply:
I don't think I quite understood what you mean here.
Do mean creating the .d.ts file?

Reply:
Maybe we aren't using these ourselves.  I remember seeing them somewhere and thought we should use them.  Something like this above each exported functions:
/**
 * Adds two numbers.
 * @param {number} num1 The first number to add.
 * @param {number} num2 The second number to add.
 * @return {number} The result of adding num1 and num2.
 */